common/src/test/resources/failover-priority-config.xml

```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" ?>
+<!--
+    Copyright Terracotta, Inc.
+    Copyright Super iPaaS Integration LLC, an IBM Company 2024
+-->
 <tc:tc-config xmlns:tc="http://www.terracotta.org/config"
```

Comment:
Since all files have either TPL or Apache license, and TPL license would be later converted to Apache license, should this file also have Apache license?

Reply:
this should be apache license, I have added the same.